applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -606,6 +606,21 @@ The current maximum number of Bluetooth LE connections that can be selected usin
 
                west build -b nrf7002dk/nrf5340/cpuapp -- -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"
 
+Configuring Bluetooth LE connection and scan parameters
+-------------------------------------------------------
+
+You can set your own Bluetooth connection parameters instead of accepting the default ones requested by the peripheral device. To do this, enable the ``BRIDGE_FORCE_BT_CONNECTION_PARAMS=y`` option, which allows you to modify the parameters.
+
+Once enabled, you can adjust the following Kconfig values to set the desired parameters:
+
+- ``BRIDGE_BT_SCAN_WINDOW``
```

Comment:
CONFIG missing in all the below. Additionally it would be nice to add explanation after the - mark. How does the user should know what do these configs mean?